<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyPlayer</class>
 <widget class="QWidget" name="MyPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyPlayer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qml.qrc">
    <normaloff>:/resources/images/MyPlayer.svg</normaloff>:/resources/images/MyPlayer.svg</iconset>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QQuickWidget" name="quickWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>600</height>
    </rect>
   </property>
   <property name="resizeMode">
    <enum>QQuickWidget::SizeRootObjectToView</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>256</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header>QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qml.qrc"/>
 </resources>
 <connections/>
</ui>
